### ISSUE TITLE
Consider adding `django-stubs` and `drf-stubs` plugins!

### ISSUE THREAD

**Issue by sobolevn** [kracekumar/python-typing-koans#14]
Hi! I see that you are already using `django-stubs` and `drf-stubs` which is awesome!

But, looks like this koan does miss plugin definitions: https://github.com/kracekumar/python-typing-koans/blob/main/koans/dj_koans/mysite/mypy.ini

Is it intended?
Thanks!

**Comment by kracekumar** [comment]
No. Since the command is run from the top-level, the mypy.ini in the top-level directory contains the plugins - https://github.com/kracekumar/python-typing-koans/blob/main/mypy.ini.

That's just kept for users to notice where to keep the plugin in the project. Will add a note to the file. 

Thanks for pointing it out.
